opm/core/transport/implicit/transport_source.c

```diff
@@ -97,6 +96,7 @@ create_transport_source(int nsrc, int nphase)
             src->pressure   = NULL;  src->flux       = NULL;
             src->saturation = NULL;  src->surfvolume = NULL;
 
+            int status;
```

Comment:
This is not pedantically correct C89 I think even though gcc accepts it. I'll not let it prevent merging, we might want to  start compiling these files with a c++ compiler anyway.